### ISSUE TITLE
Game Crash after interacting with bookshelf

### ISSUE THREAD

**Issue by cosycloud** [An-Sar/PrimalStorage#1]
Created a new shelf, and placed it, but when I went to interact with it, my game crashed. 

Launcher Error: 
```
The game crashed whilst unexpected error
Error: java.lang.IllegalAccessError: class nmd.primalstorage.helpers.ItemHelper tried to access private method 'void net.minecraft.class_1799.method_7957()' (nmd.primalstorage.helpers.ItemHelper and net.minecraft.class_1799 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @62e136d3)
```
Crash Report

```
---- Minecraft Crash Report ----
// My bad.

Time: 2023-05-18 17:28:00
Description: Unexpected error

java.lang.IllegalAccessError: class nmd.primalstorage.helpers.ItemHelper tried to access private method 'void net.minecraft.class_1799.method_7957()' (nmd.primalstorage.helpers.ItemHelper and net.minecraft.class_1799 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @62e136d3)
	at nmd.primalstorage.helpers.ItemHelper.loadStack(ItemHelper.java:168)
	at nmd.primalstorage.helpers.ItemHelper.loadAllItems(ItemHelper.java:210)
	at nmd.primalstorage.blocks.entities.ShelvesEntity.method_11014(ShelvesEntity.java:40)
	at net.minecraft.class_634.method_38542(class_634.java:1138)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at net.minecraft.class_634.method_11094(class_634.java:1135)
	at net.minecraft.class_2622.method_11292(class_2622.java:49)
	at net.minecraft.class_2622.method_11054(class_2622.java:14)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:157)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:131)
	at net.minecraft.class_1255.method_5383(class_1255.java:116)
	at net.minecraft.class_310.method_1523(class_310.java:1129)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at nmd.primalstorage.helpers.ItemHelper.loadStack(ItemHelper.java:168)
	at nmd.primalstorage.helpers.ItemHelper.loadAllItems(ItemHelper.java:210)
	at nmd.primalstorage.blocks.entities.ShelvesEntity.method_11014(ShelvesEntity.java:40)
	at net.minecraft.class_634.method_38542(class_634.java:1138)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at net.minecraft.class_634.method_11094(class_634.java:1135)
	at net.minecraft.class_2622.method_11292(class_2622.java:49)
	at net.minecraft.class_2622.method_11054(class_2622.java:14)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:157)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:131)

-- Affected level --
Details:
	All players: 1 total; [class_746['cosycloud'/172, l='ClientLevel', x=3122.09, y=63.00, z=7433.01]]
	Chunk stats: 1024, 609
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,90,0), Section: (at 0,10,0 in 0,5,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 764443 game time, 1372079 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:453)
	at net.minecraft.class_310.method_1587(class_310.java:2341)
	at net.minecraft.class_310.method_1514(class_310.java:792)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, lambdabettergrass/default, Immersive Weathering Generated Pack

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1097617376 bytes (1046 MiB) / 6341787648 bytes (6048 MiB) up to 6341787648 bytes (6048 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i5-12400F
	Identifier: Intel64 Family 6 Model 151 Stepping 5
	Microarchitecture: unknown
	Frequency (GHz): 2.50
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: Virtual Desktop Monitor
	Graphics card #0 vendor: Virtual Desktop, Inc.
	Graphics card #0 VRAM (MB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: DriverVersion=15.39.56.845
	Graphics card #1 name: NVIDIA GeForce RTX 3060
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 4095.00
	Graphics card #1 deviceId: 0x2504
	Graphics card #1 versionInfo: DriverVersion=31.0.15.2824
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 59205.43
	Virtual memory used (MB): 28293.54
	Swap memory total (MB): 43008.00
	Swap memory used (MB): 2222.97
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx6048m -Xms256m
	Fabric Mods: 
		accelerateddecay: Accelerated Decay 1.0.0+mc1.19.2
		adorn: Adorn 3.8.3+1.19.2
		ae2: Applied Energistics 2 12.9.3
			noindium: No Indium? 1.1.0+1.19
		ambientsounds: Ambient Sounds (Fabric) 5.2.13
		appleskin: AppleSkin 2.4.1+mc1.19
		aquatictorches: Aquatic Torches 1.0.0
		architectury: Architectury 6.5.82
		artifacts: Artifacts 7.1.1+fabric
			expandability: ExpandAbility 6.0.0
			step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		balm-fabric: Balm 4.5.7
		base_camp: Base Camp 1.4.5
		beautify: Beautify 1.1.1+fabric-1.19.2
			cardinal-components-base: Cardinal Components API (base) 5.0.2
			cardinal-components-entity: Cardinal Components API (entities) 5.0.2
			cardinal-components-world: Cardinal Components API (worlds) 5.0.2
		betteradvancements: Better Advancements 0.2.2.142
		betteranimalsplus: Better Animals Plus 1.19.2-11.0.10
		betterf3: BetterF3 4.0.0
		bewitchment: Bewitchment 1.19-7
			fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
			impersonate: Impersonate 2.8.0
			playerabilitylib: Pal 1.6.0
			reach-entity-attributes: Reach Entity Attributes 2.3.0
			terraform-wood-api-v1: Terraform Wood API (v1) 4.2.0
		bookshelf: Bookshelf 16.2.18
		botanypots: BotanyPots 9.0.33
		botanypotstiers: BotanyPotsTiers 3.3.2
		botanytrees: BotanyTrees 5.0.8
		bushierflowers: Bushier Flowers 0.0.1-1.19.2
		bwplus: Bewitchment Plus 1.19-10
		byg: Oh The Biomes You'll Go 2.0.0.13
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
		cacao: Croptopia's Chocolaterie 1.2
		cardinal-components: Cardinal Components API 5.0.2
			cardinal-components-block: Cardinal Components API (blocks) 5.0.2
			cardinal-components-chunk: Cardinal Components API (chunks) 5.0.2
			cardinal-components-item: Cardinal Components API (items) 5.0.2
			cardinal-components-level: Cardinal Components API (world saves) 5.0.2
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 5.0.2
		carryon: Carry On 2.0.5.11
			com_github_llamalad7_mixinextras: MixinExtras 0.1.1
		charm: Charm 4.4.4
			com_moandjiezana_toml_toml4j: toml4j 0.7.2
		chiselsandbits: chisels-and-bits 1.3.135
			org_lz4_lz4-pure-java: lz4-pure-java 1.8.0
			saeculariacaudices: Saecularia Caudices 1.0.16
			scena: Scena 1.0.95
		climbladdersfast: ClimbLaddersFast 3.0.6-1.19
		cloth-config: Cloth Config v8 8.2.88
			cloth-basic-math: cloth-basic-math 0.6.1
		collective: Collective 6.53
		comforts: Comforts 6.0.4+1.19.2
			spectrelib: SpectreLib 0.12.4+1.19.2
		compact_storage: Compact Storage 5.0.3_1.19.2
		controlling: Controlling For Fabric 10.0+7
		cookingforblockheads: Cooking for Blockheads 13.3.2
		corgilib: CorgiLib 1.0.0.33
		creativecore: CreativeCore (Fabric) 2.9.3
		croptopia: Croptopia 2.2.2
			com_typesafe_config: config 1.4.1
			io_leangen_geantyref_geantyref: geantyref 1.3.11
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		croptosis: Croptosis 1.5.1
		darkutils: DarkUtilities 13.1.7
		dataattributes: Data Attributes 1.4.4+1.19.2
		decorative_blocks: Decorative Blocks 3.0.0
		diggusmaximus: Diggus Maximus 1.5.8-1.19
			kyrptconfig: Kyrpt Config 1.4.14-1.19
		display_case: Display Case Mod 1.0.3
		doubledoors: Double Doors 4.5
		easierchests: EasierChests 1.19.1-fabric0.58.5-1.7.0
			crowdin-translate: CrowdinTranslate 1.4+1.19
			gbfabrictools: GBfabrictools 1.3.4+1.19
		ecologics: Ecologics 2.1.11
		effective: Effective 1.4
		enchantedtooltips: Enchanted ToolTips 1.3.7-1.19
		enchantinginfuser: Enchanting Infuser 4.2.2
		enchdesc: EnchantmentDescriptions 13.0.14
		equipmentcompare: Equipment Compare 1.3.1
		expandeddelight: Expanded Delight 0.2.5.2
		extended_drawers: Extended Drawers 1.4.0+mc.1.19.2
			graphlib: Graph Lib 0.3.6+1.19.2
			mconfig: Mconfig 1.2.0+mc.1.19-rc1
				de_poiu_apron_apron: apron 2.1.1
		fabric-api: Fabric API 0.76.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.15+8f4e8eb390
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.14+93d8cb8290
			fabric-biome-api-v1: Fabric Biome API (v1) 9.1.1+16f1e31390
			fabric-block-api-v1: Fabric Block API (v1) 1.0.2+e415d50e90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.25+cafc6e8e90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.5+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.16+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.2.1+413cbbc790
			fabric-commands-v0: Fabric Commands (v0) 0.2.33+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.41+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.5.2+7c6cd14d90
			fabric-convention-tags-v1: Fabric Convention Tags 1.3.0+4bc6e26290
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.8+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.3.8+413cbbc790
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.35+0d0f210290
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.4+9244241690
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.34+562bff6e90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.36+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.24+b6b6abb490
			fabric-item-api-v1: Fabric Item API (v1) 1.6.6+b7d1888890
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.39+9244241690
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.25+5c4fce2890
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.23+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.4+1b46dc7890
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.13+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.16+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.7+93d8cb8290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.24+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.21+c6af733c90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.11+10eb22f490
			fabric-networking-v0: Fabric Networking (v0) 0.3.28+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.2.2+d8ef690890
			fabric-particles-v1: Fabric Particles (v1) 1.0.14+4d0d570390
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.1+413cbbc790
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.32+9244241690
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.2.1+1adbf27790
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.8.0+1adbf27790
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.24+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.19+6e0787e690
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.11+4d0d570390
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.27+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.12.1+d8ef690890
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.1.2+aae9039d90
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.8.4+edbdcddb90
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.32+4d0d570390
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.7+1cc24b1b90
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.2+c4f28df590
			fabric-textures-v0: Fabric Textures (v0) 1.0.24+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.6+413cbbc790
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.3+08b73de490
		fabric-language-kotlin: Fabric Language Kotlin 1.9.4+kotlin.1.8.21
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.8.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.8.21
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.20.2
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.5.0
		fabricloader: Fabric Loader 0.14.19
		farmersdelight: Farmer's Delight 1.19.2-1.3.9
		farmersknives: Farmer's Knives 2.4
		farmersrespite: Farmer's Respite 2.2.4
			porting_lib: Porting Lib 2.1.656+1.19.2
				forge_tags: Porting Lib Tags 3.0
				porting_lib_base: Porting Lib Base 2.1.656+1.19.2
					mm: Manningham Mills 2.3
					serialization_hooks: Serialization Hooks 0.3.26
				porting_lib_entity: Porting Lib Entity 2.1.656+1.19.2
				porting_lib_lazy_registration: Porting Lib Lazy Register 2.1.656+1.19.2
				porting_lib_model_generators: Porting Lib Model Generators 2.1.656+1.19.2
				porting_lib_models: Porting Lib Models 2.1.656+1.19.2
				porting_lib_networking: Porting Lib Networking 2.1.656+1.19.2
				porting_lib_obj_loader: Porting Lib Obj Loader 2.1.656+1.19.2
					porting_lib_model_loader: Porting Lib Model Loader 2.1.656+1.19.2
				porting_lib_transfer: Porting Lib Transfer 2.1.656+1.19.2
					porting_lib_extensions: Porting Lib Extensions 2.1.656+1.19.2
						porting_lib_accessors: Porting Lib Accessors 2.1.656+1.19.2
						porting_lib_attributes: Porting Lib Attributes 2.1.656+1.19.2
							porting_lib_constants: Porting Lib Constants 2.1.656+1.19.2
						porting_lib_common: Porting Lib Common 2.1.656+1.19.2
		fdjei: Just Enough Farmer's Recipes 1.0.1
		forgeconfigapiport: Forge Config API Port 4.2.11
		friendlyfire: FriendlyFire 14.0.4
		geckolib3: Geckolib 3.1.40
			com_eliotlash_mclib_mclib: mclib 20
		guardvillagers: Guard Villagers Fabric 1.19.2-1.1.2
		harvestwithease: Harvest with ease 6.0.1.2-final
		healingcampfire: Healing Campfire 5.1
		iceberg: Iceberg 1.0.46
		illuminations: Illuminations 1.10.11
		immersive_weathering: Immersive Weathering 1.19.2-1.2.9
		indium: Indium 1.0.9+mc1.19.2
		inventorysorter: Inventory Sorter 1.8.10-1.19
		iris: Iris 1.6.2
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre13
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		itemphysic: ItemPhysic (Fabric) 1.6.6
		jade: Jade 8.7.3
		jadeaddons: Jade Addons 3.1.0
		java: OpenJDK 64-Bit Server VM 17
		jei: Just Enough Items 11.6.0.1015
		jeresources: Just Enough Resources 1.2.2.200
		labels: labels 1.19.2-1.9
		lambdabettergrass: LambdaBetterGrass 1.3.0+1.19
		lambdynlights: LambDynamicLights 2.2.0+1.19.2
			pride: Pride Lib 1.1.2+1.19
			spruceui: SpruceUI 4.1.0+1.19.2
		lightoverlay: Light Overlay 7.0.0
		macawsbridgesbyg: Macaw's Bridges - BYG 1.19.2-1.2
		magnumtorch: Magnum Torch 4.2.2
		malilib: MaLiLib 0.13.0
		mcwbridges: Macaw's Bridges 2.0.7
		mcwdoors: Macaw's Doors 1.0.8
		mcwfences: Macaw's Fences and Walls 1.0.7
		mcwfurnitures: Macaw's Furniture 3.1.0
		mcwlights: Macaw's Lights and Lamps 1.0.5
		mcwpaths: Macaw's Paths and Pavings 1.0.2
		mcwwindows: Macaw's Windows 2.1.2
		midnightlib: MidnightLib 1.0.0
		minecraft: Minecraft 1.19.2
		modmenu: Mod Menu 4.2.0-beta.2
		moonlight: Moonlight 1.19.2-2.2.33
		mousetweaks: Mouse Tweaks 2.22
		naturalist: Naturalist 3.0.2
		naturescompass: Nature's Compass 1.19.2-2.1.0-fabric
		nets: Not Enough Trinket Slots (NETS) 0.3.5.2
		notenoughanimations: NotEnoughAnimations 1.6.2
		oaks_nature: Oak's Nature 0.2.1 fabric
		owo: oωo 0.8.5+1.19
			blue_endless_jankson: jankson 1.2.1
		patchouli: Patchouli 1.19.2-77-FABRIC
			fiber: fiber 0.23.0-2
		pehkui: Pehkui 3.7.3+1.14.4-1.20
			kanos_config: Kanos Config 0.4.0+1.14.4-1.19.4
		pickablevillagers: Pickable Villagers 1.3.4+1.19.2
		pickupnotifier: Pick Up Notifier 4.2.4
		pigpen: PigPen 11.1.2
		playerex: PlayerEx 3.5.2
			net_objecthunter_exp4j: exp4j 0.4.8
			opc: Offline Player Cache 0.6.4
			placeholder-api: Placeholder API 2.0.0-pre.1+1.19.2
		polymorph: Polymorph 0.46.1+1.19.2
		powah: Powah! 4.0.10
		prefab: Prefab 4.2.4
		primalstorage: PrimalStorage 1.1
		puzzleslib: Puzzles Lib 4.4.0
		reborncore: Reborn Core 5.4.0
			team_reborn_energy: Energy 2.3.0
		resourceful_tools: Resourceful Tools 1.3.26
			pugh_lib: Pugh Lib 1.4.15
		runelic: Runelic 14.1.4
		satin: Satin 1.9.0
		sit: Sit 1.19-20
		sod: Sod 22.10.13
		sodium: Sodium 0.4.4+build.18
			org_joml_joml: joml 1.10.4
		soulshards: Soul Shards Despawn 1.0.5
		storage-bookshelves: Storage Bookshelves 0.0.1+1.19.2
		structory: Structory 1.0.1
		techreborn: Tech Reborn 5.4.0
		terrablender: TerraBlender 2.0.1.136
		things: Things 0.3.0+1.19
		toolstats: ToolStats 12.0.2
		tooltipfix: ToolTip Fix 1.1.1-1.19
		tr-patchouli: Tech Reborn Patchouli 2.1.1
		travelersbackpack: Traveler's Backpack 1.19.2-8.2.22
		trinkets: Trinkets 3.4.2
		tweakeroo: Tweakeroo 0.14.3
		villagernames: Villager Names 5.2
		vinery: Vinery 1.2.5
		wildlife: Wildlife 1.19.2-2.3
		xaerominimap: Xaero's Minimap 23.4.3
		xaeroworldmap: Xaero's World Map 1.30.2
		xp_storage: XP Storage 1.4.3+1.19
		yigd: You're in Grave Danger 1.4.9
		yungsapi: YUNG's API 1.19.2-Fabric-3.8.9
			org_javassist_javassist: javassist 3.28.0-GA
			org_reflections_reflections: reflections 0.10.2
		yungsbridges: YUNG's Bridges 1.19.2-Fabric-3.1.0
		z_mcwfencesbyg: Macaw's Fences - BYG 1.19.2-1.0
	Loaded Shaderpack: (off)
	NEC status: No NEC detected
	Launched Version: fabric-loader-0.14.19-1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce RTX 3060/PCIe/SSE2 GL version 3.2.0 NVIDIA 528.24, NVIDIA Corporation
	Window size: 1920x1009
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, Fabric Mods, lambdabettergrass/default, Immersive Weathering Generated Pack
	Current Language: English (US)
	CPU: 12x 12th Gen Intel(R) Core(TM) i5-12400F
	Server Running: true
	Player Count: 1 / 8; [class_3222['cosycloud'/172, l='ServerLevel[The Land of Cosy]', x=3122.09, y=63.00, z=7433.01]]
	Data Packs: vanilla, Fabric Mods, minecraft/croptop, Immersive Weathering Generated Pack
	World Generation: Stable
```

**Comment by An-Sar** [comment]
this will be fixed in the next release

**Comment by An-Sar** [comment]
this is now fixed in 1.2, published just now to both modrinth and curse

**Comment by cosycloud** [comment]
Excellent! Just tried it out - working great now thanks!
